### PR TITLE
fix(tooltip): handle esc keydown event

### PR DESCRIPTION
**Pull request by dakahn**
adds a check in the `handleKeyPress` method that closes the tooltip if escape is pressed.

ref #2076 

### to test
open the tool tip and press escape. the tooltip should close.

### CODE REVIEW

File: src/components/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -374,6 +374,11 @@ class Tooltip extends Component {
   handleKeyPress = evt => {
     const key = evt.key || evt.which;
 
+    if (key === 'Escape' || key === 27) {
```
Comment (asudoh) [inline src/components/Tooltip/Tooltip.js]:
```suggestion
    if (key === 'Escape' || key === 'Esc' || key === 27) {
```

Talking with @vpicone revealed that `evt.key` yields `Esc` in IE and looks that we should cover that.

Comment (emyarod) [reply src/components/Tooltip/Tooltip.js]: would we want to also update and use these helper functions https://github.com/IBM/carbon-components-react/blob/master/src/tools/key.js?

Comment (asudoh) [reply src/components/Tooltip/Tooltip.js]: @emyarod Good point - I think it makes sense to me.

**Comment by asudoh** [comment]
Thanks @dakahn for the update and my apologies for being late on this - Updated code basically looks good to me, could you please look at the test failure?

**Comment by asudoh** [comment]
The updated code LGTM 👍 - Thanks @dakahn!

**Pull request merged**